sdk/storage/azure-storage-extensions/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Release History
 
+This version and all future versions will require Python 3.9+. Python 3.8 is no longer supported.
```

Comment:
This is what I mentioned we don't need for extensions as extensions has not yet been released.